packages/editor/src/components/post-locked-modal/index.js

```diff
@@ -55,6 +57,7 @@ function PostLockedModal() {
 			activePostLock: getActivePostLock(),
 			postType: getPostType( getEditedPostAttribute( 'type' ) ),
 			previewLink: getEditedPostPreviewLink(),
+			supportsSync: getEntityConfig( 'postType', entityName )?.syncConfig?.enabled,
```

Comment:
We removed the `enabled` property in favor of presence of `syncConfig`
```suggestion
			supportsSync: Boolean( getEntityConfig( 'postType', entityName )?.syncConfig ),
```